tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -566,51 +572,47 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "bb23fba3",
+   "id": "c60ed72a",
    "metadata": {},
    "outputs": [],
    "source": [
-    "wandb_api_key = \"apikey\"\n",
-    "wandb_project_name = \"GermanTTS\"\n",
-    "wandb_run_name = \"tutorial\"\n",
-    "\n",
-    "!wandb login ${wandb_api_key}"
+    "!wandb login #paste_wandb_apikey_here"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "48e16fed",
+   "id": "51ec332e",
    "metadata": {},
    "source": [
-    "Now we are ready for training our model! Let's try to train FastPitch."
+    "Now we are ready for training our model! Let's try to train FastPitch. Paste the PITCH_MEAN and PITCH_STD from previous steps here."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "1b37ec68",
+   "id": "4ad43d30",
    "metadata": {},
    "outputs": [],
    "source": [
-    "!(cd /NeMo && CUDA_VISIBLE_DEVICES=0 python fastpitch.py --config-path /NeMo --config-name fastpitch_align_22050 \\\n",
+    "!(cd NeMoGermanTTS && CUDA_VISIBLE_DEVICES=0 python fastpitch.py --config-path . --config-name fastpitch_align_22050 \\\n",
     "    model.train_ds.dataloader_params.batch_size=32 \\\n",
     "    model.validation_ds.dataloader_params.batch_size=32 \\\n",
-    "    train_dataset=/Data/thorsten-de/train_manifest_phonemes.json \\\n",
-    "    validation_datasets=/Data/thorsten-de/val_manifest_phonemes.json \\\n",
-    "    sup_data_path=/Data/thorsten-de/phonemes/ \\\n",
-    "    whitelist_path=/NeMo/whitelist.tsv \\\n",
-    "    exp_manager.exp_dir=/result \\\n",
+    "    train_dataset=../DataGermanTTS/thorsten-de/train_manifest_phonemes.json \\\n",
+    "    validation_datasets=../DataGermanTTS/thorsten-de/val_manifest_phonemes.json \\\n",
+    "    sup_data_path=../DataGermanTTS/thorsten-de/phonemes/ \\\n",
+    "    whitelist_path=./whitelist.tsv \\\n",
+    "    exp_manager.exp_dir=resultGermanTTS \\\n",
     "    trainer.max_epochs=1 \\\n",
-    "    pitch_mean=132.524658203125 \\\n",
-    "    pitch_std=37.389366149902344 \\\n",
+    "    pitch_mean=#paste_pitch_mean_here \\\n",
+    "    pitch_std=#paste_pitch_std_here \\\n",
```

Comment:
Would be cleaner to have the pitch mean/std be python variables to be overridden

Reply:
Thanks for the feedback Ryan. Will improve in next iterations.